mixer/cmd/mixs/cmd/probe.go

```diff
@@ -31,7 +31,7 @@ func probeCmd(printf, fatalf shared.FormatFn) *cobra.Command {
 		Args:  cobra.ExactArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
 			if !probeOptions.IsValid() {
-				fatalf("Some options are not valid")
+				fatalf("--probe-path should be provided and --interval should be longer than 0s")
```

Comment:
Should we consider adding something like:

```
func (o *Options) ValidityErrors() string {
   s := ""
   if o.Path = "" {
      s = "Path must be provided. "
   }
   if o.UpdateInterval <= 0*time.Second {
      s += "Interval must be longer than 0s."
   }
}
```

And then just have this be: `fatalf(probeOptions.ValidityErrors())`

Then we wouldn't have to try and update these error messages all over the codebase each time the probe options change.